content/en/blog/2025/sail-operator-ga/index.md

```diff
@@ -0,0 +1,302 @@
+---
+
+The Sail operator is GA with a clear mission: to simplify and streamline Istio management in your cluster. By using Kubernetes operator practices, Sail operator aims to add new value to istio by improving the upgrade process.
+
+## Simplified Deployment & Management
+
+The Sail operator is engineered to cut down the complexity of installing and running Istio with an operator. It automates manual tasks, ensuring a consistent, reliable, and uncomplicated experience from initial installation to ongoing maintenance and upgrades of Istio versions in your cluster. Besides this, the Sail operator APIs are built around Istio’s Helm chart APIs, which means that all the istio configurations are available through the Sail Operator CRDs’ values.
+
+We encourage users to go through our live [documentation](https://github.com/istio-ecosystem/sail-operator/tree/main/docs) to help you easily get deep into this new way to manage your istio control plane.
+
+The main resources that are part of the Sail operator are:
+* `Istio`: manages your istio control plane.
+* `IstioRevision`: it represents a revision of the control plane.
+* `IstioRevisionTag`: resource represents a stable Revision Tag, which functions as an alias for Istio control plane revisions.
+* `IstioCNI`: Istio's CNI plugin resource.
+* `ZTunnel`: ambient mode Ztunnel DaemonSet (alpha feature).
+
+Note that if you are migrating from the [deprecated and deleted] In-Cluster Istio operator, you can check this section in our [documentation](https://github.com/istio-ecosystem/sail-operator/tree/main/docs#migrating-from-istio-in-cluster-operator) where we explain the equivalence of resources, or you can try also our [resource converter](https://github.com/istio-ecosystem/sail-operator/tree/main/docs#converter-script) to convert your  IstioOperator resource to an Istio resource easily.
+
+## Main Features and support
+
+- Support for multiple Istio versions.
+- Two update strategies are supported: `InPlace` and `RevisionBased`. Check our documentation for more information about the update types supported.
+- Support for multicluster Istio [deployment models](/docs/setup/install/multicluster/): multi-primary, primary-remote, external control plane. More information and examples in our [documentation](https://github.com/istio-ecosystem/sail-operator/blob/main/docs/README.md#multi-cluster).
+- Support for [multiple mesh](/docs/ops/deployment/deployment-models/#multiple-meshes): The Sail Operator supports running multiple meshes on a single cluster and associating each workload with a specific mesh. Each mesh is managed by a separate control plane. More information and examples can be found [here](https://github.com/istio-ecosystem/sail-operator/blob/main/docs/README.md#multiple-meshes-on-a-single-cluster).
+- [Dual-Stack](https://github.com/istio-ecosystem/sail-operator/blob/main/docs/README.md#dual-stack-support) support since istio `v1.23` version.
```

Comment:
Again I don't think this is really a Sail Operator feature since all it is doing is passing through the helm values.